frame/contracts/src/tests.rs

```diff
@@ -4256,12 +4225,15 @@ fn set_code_extrinsic() {
 #[test]
 fn slash_cannot_kill_account() {
```

Comment:
do we need this test at all as slashing can not kill account by design now that we moved to fungible?

Reply:
In general I would agree, but the fact that this extra test is there does not hurt, and it adds an extra shield that can potentially prevent a situation in which we are not thinking of at the moment. (It might sound stupid, but maybe a situation where the Currency type used in the test does not honour the actual definition of the trait implementation)